### ISSUE TITLE
Media & Text block shouldn't default to large font size - especially when font sizes are disabled

### ISSUE THREAD

**Issue by smerriman** [WordPress/gutenberg#21126]
Using a theme that contains:

`add_theme_support('disable-custom-font-sizes');
add_theme_support('editor-font-sizes',array());`

When a Media & Text block is added, the default paragraph for the content has a `has-large-font-size` class. While it doesn't appear large in the backend, it still appears large in the frontend.

It's possible that the above code should remove the default styling of has-large-font-size from the frontend like it does in the backend - but regardless of that, the class shouldn't be added to the paragraph tag.

**Desktop (please complete the following information):**
 - OS: Windows 7 (but applies to all)
 - Browser: Firefox (but applies to all)

WordPress 5.3.2.

**Comment by talldan** [comment]
I was able to reproduce this. The cause seems to be that the fontSize is hardcoded in the block template:
https://github.com/WordPress/gutenberg/blob/ef799f64548ac9a373a90908376c79c57955f948/packages/block-library/src/media-text/edit.js#L38-L46

I think either the template should have a sprinkling of magic so that it can detect when the theme disables font sizes, or the font size system should be smart enough not to apply the class.

**Comment by JoshuaDoshua** [comment]
UX note: because this is a paragraph block, you can't quickly transform to a paragraph (bc it's already a p) from what _may appear to be a heading block_, end users can be confused.

+1 to remove `fontSize` from the template

**Comment by fembuelita** [comment]
Using @smerriman's code of

```php
add_theme_support('disable-custom-font-sizes'); 
add_theme_support('editor-font-sizes', []);
```

in your functions.php (or wherever makes sense) in combination with your style-editor.css file:

```css
.editor-styles-wrapper .has-small-font-size,
.editor-styles-wrapper .has-normal-font-size,
.editor-styles-wrapper .has-medium-font-size,
.editor-styles-wrapper .has-large-font-size,
.editor-styles-wrapper .has-huge-font-size {
    font-size: initial !important;
}
```

**Does** work as a temporary work-around on this issue. I think it would be preferable more eloquently override/change/disable default settings for a single component as this is a "brute-force" approach that modifies all Gutenberg components that normally support typography size selections.

**Comment by sabernhardt** [comment]
[Trac #53366](https://core.trac.wordpress.org/ticket/53366) has another request to use the "default" font size for these paragraphs.

**Comment by sabernhardt** [comment]
Could it be as simple as removing the one line from gutenberg/packages/block-library/src/media-text/edit.js?

**Comment by ekazda** [comment]
How is this still an issue? It is ridiculous that Gutenberg is riddled with issues like this where settings are simply applied to blocks and elements that cannot be properly managed or removed without hacks to the system. Please start fixing bugs like this and properly documenting how to manage settings via the theme.json.

**Comment by Humanify-nl** [comment]
Walked into this bug today.

Media-text has for me always been a weird block. It appears nice to have, but it pretty much isn't anymore with all the flex-controls we have in the columns block. To be fair, this block is becoming fairly redundant since we can build the same with a column or pattern (or reusable block for all I care). 

The 'crop fill to column' for the image is nice but it's not entirely needed.

I can see reasons in above why this fix is ignored, because it may be on track to become removed / replaced possibly?

**Comment by carolinan** [comment]
This was fixed in #40329.
